src/mds/agg_mds/adapters.py

```diff
@@ -914,7 +916,7 @@ def getRemoteDataAsJson(self, **kwargs) -> Dict:
                     url += f"&{filters}"
                 if field_name is not None and field_value is not None:
                     url += f"&{guid_type}.{field_name}={field_value}"
-                response = httpx.get(url)
+                response = httpx.get(url, timeout=60)
```

Comment:
what was the issue exactly? was the default timeout (i think 5s) not enough?

Reply:
Yeah! MIDRC takes more than 5 to 10 seconds to respond. This is because the data present in MIDRC is too huge to response. Link -->  [here](https://data.midrc.org/mds/metadata?data=true&_guid_type=discovery_metadata&offset=0&limit=1000)